core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,30 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
```

Comment:
> And if the section number is added to the javadoc, who updates the javadoc in the future versions of the code when the manual changes?

Thats a good point.  We could be vauge about it and just say see the user manual for more information.   Also since its not a URL, we can not run tools to check for bad links.

> I would think that if you want to use the class loader context feature, then you have probably already read the section in the manual to understand how to configure it.

I think its reasonable to assume that someone will start using 1.8 w/o ever reading the user manual and notice this new method in an IDE.  When they see the new method in the IDE, currently it would not have much javadocs.  However there are nice docs elsewhere. It would be nice to make the user aware of those docs and help them find them.  Personally I think helping guide the user in this case is better than doing nothing because of the valid concern that info pointed to may move.